src/DiscordBot.py

```diff
@@ -10,23 +10,33 @@
 - A Discord bot token with the necessary permissions.
 - A GitHub personal access token with the necessary permissions.
 """
-from discord.ext import tasks, commands
+import logging
+import os
+from datetime import datetime
+
 import discord
+from discord.ext import commands, tasks
+from dotenv import load_dotenv
 from GitHubUtilities import GitHubUtilities
 from InternshipUtilities import InternshipUtilities
-import os
-from dotenv import load_dotenv
-from datetime import datetime
-import traceback
 
 load_dotenv()
 DISCORD_TOKEN = os.getenv("DISCORD_TOKEN")
 CHANNEL_ID = os.getenv("CHANNEL_ID")
 GITHUB_TOKEN = os.getenv("GIT_TOKEN")
 
+logging.basicConfig(
```

Comment:
Can you please add some comments here about the set up

Reply:
Yessir